.github/workflows/check-catalyst.yaml

```diff
@@ -289,14 +268,12 @@ jobs:
 
       - name: Build Catalyst-Runtime
         run: |
-          cmake -S runtime -B runtime-build -G Ninja \
-                -DCMAKE_BUILD_TYPE=Release \
-                -DCMAKE_C_COMPILER=clang \
-                -DCMAKE_CXX_COMPILER=clang++ \
-                -DLIGHTNING_GIT_TAG=$LIGHTNING_GIT_TAG_VALUE \
-                -DQIR_STDLIB_LIB="$(pwd)/runtime/qir-stdlib-build/" \
-                -DQIR_STDLIB_INCLUDES="$(pwd)/runtime/qir-stdlib-build/include/"
-          cmake --build runtime-build --target rt_capi -j$(nproc)
+          pushd runtime
+          RT_BUILD_DIR=../runtime-build \
+          LIGHTNING_GIT_TAG_VALUE=$LIGHTNING_GIT_TAG_VALUE \
```

Comment:
Is this line not redundant? Or does it not just reassign an env variable to itself?
```suggestion
```